pkgs/development/node-packages/aliases.nix

```diff
@@ -41,9 +41,9 @@ mapAliases {
   "@antora/cli" = pkgs.antora; # Added 2023-05-06
   "@githubnext/github-copilot-cli" = pkgs.github-copilot-cli; # Added 2023-05-02
   "@google/clasp" = pkgs.google-clasp; # Added 2023-05-07
-  "@nestjs/cli" = pkgs.nest-cli; # Added 2023-05-06
```

Comment:
I think we sorted that first because it starts with the special character

Reply:
Reverted.